readthedocs/templates/search/search.html

```diff
@@ -14,7 +14,7 @@
               </div>
               <div class="submit-input-wrapper">
                 {% comment %}Translators: This is about starting a search{% endcomment %}
-                <input type="submit" value="{% trans "Go crazy." %}">
+                <input type="submit" value='{% trans "Search" %}'
```

Comment:
I think you removed the `>` accidentally here

Reply:
oops.. my bad :D I'll correct it for sure :)